### PR TITLE
Add YMCA Website Services to Example Sites

### PR DESCRIPTION
**Pull request by froboy**
Thanks for all of your hard work!

### CODE REVIEW

File: userguide/content/en/docs/examples/_index.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Example sites that have low to no customization:
 | [protobuf.dev](https://protobuf.dev) | https://github.com/protocolbuffers/protocolbuffers.github.io |
 | [Layer5 Docs](https://docs.layer5.io/) | https://github.com/layer5io/docs |
 | [Apache Parquet](https://parquet.apache.org/) | https://parquet.apache.org |
+| [YMCA Website Services - a Drupal Distribution](https://ds-docs.y.org/) | https://github.com/YCloudYUSA/yusaopeny_docs |
```
Comment (chalin) [inline userguide/content/en/docs/examples/_index.md:37]: Might you be open to shortening this?

```suggestion
| [YMCA Website Services](https://ds-docs.y.org/) | https://github.com/YCloudYUSA/yusaopeny_docs |
```

**Review by chalin — COMMENTED** [review]
Thanks @froboy, LGTM. See inline comment for suggested simplification.

File: userguide/content/en/docs/examples/_index.md (hunk 1 above)
Comment (froboy) [reply userguide/content/en/docs/examples/_index.md]: Sure. Thanks for the review!

**Comment by froboy** [comment]
Pretty sure the failing tests are unrelated to this change...

**Review by chalin — APPROVED** [review]

**Comment by chalin** [comment]
The Windows build failure has nothing to do with this PR. Merging.

**Pull request merged**